src/components/experiment-tracking/run-dataset/run-dataset.js

```diff
@@ -44,41 +45,54 @@ const RunDataset = ({
   isSingleRun,
   pinnedRun,
   selectedRunIds,
-  trackingData = [],
+  trackingData,
 }) => {
   return (
     <div
       className={classnames('details-dataset', {
         'details-dataset--single': isSingleRun,
       })}
     >
-      {trackingData.map((dataset) => {
-        const { data, datasetName } = dataset;
-
+      {Object.keys(trackingData).map((group) => {
```

Comment:
Curious how the order of the sections `Metrics` vs. `JSON data` is determined here. Is it just automatically the same as the order specified in `query getRunTrackingData`?

Reply:
query has it :)